scripts/hybrid-custody/get_accessible_proxy_account_nfts.cdc

```diff
@@ -0,0 +1,78 @@
+import "HybridCustody"
+import "NonFungibleToken"
+import "MetadataViews"
+
+
+// This script iterates through a parent's child accounts, 
+// identifies private paths with an accessible NonFungibleToken.Provider, and returns the corresponding typeIds
+pub fun main(addr: Address):AnyStruct {
+  let account = getAuthAccount(addr)
```

Comment:
We have `acct` and `account` in this script, maybe we could rename `account` to something like `parent`?

Reply:
Going to remove ln 9, we don't use it anywhere